File: uri-judge/[1144]Logical_Sequence.cpp
//Author: Hérson Reis Rezende dos Santos
//Site: hrezend.github.io

#include <bits/stdc++.h>
#define desync ios_base::sync_with_stdio(false);cin.tie(NULL);cout.tie(NULL)
using namespace std;

int main(){
    desync;
    int N;
    cin >> N;

    for(int i = 1; i <= N; i++){
        cout << i << " " << i*i << " " << i*i*i << endl;
        cout << i << " " << (i*i)+1 << " " << (i*i*i)+1 << endl;
    }
}

File: uri-judge/[1144]Logical_Sequence.java
//Author: Hérson Reis Rezende dos Santos
//Site: hrezend.github.io

import java.util.Scanner;
 
public class Main {
 
    public static void main(String[] args){
 
        Scanner sc = new Scanner(System.in);
        int N;
        N = sc.nextInt();

        for(int i = 1; i <= N; i++){
            System.out.println(i + " " + (i*i) + " " + (i*i*i));
            System.out.println(i + " " + ((i*i)+1) + " " + ((i*i*i)+1));
        }
 
    }
 
}
